<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtzManageDialog</class>
 <widget class="QDialog" name="PtzManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>678</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,1,0">
      <item>
       <widget class="QWidget" name="buttonsWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="savePositionButton">
           <property name="text">
            <string>Save Current Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="goToPositionButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Go To Position</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addTourButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Create Tour</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startTourButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start Tour</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QnTableView" name="tableView">
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="tourGroupBox">
           <property name="title">
            <string>Details</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="tourStackedWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="tourPage">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QnPtzTourWidget" name="tourEditWidget" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="noTourPage">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>No Tour Selected</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="previewGroupBox">
           <property name="title">
            <string>Preview</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QStackedWidget" name="previewStackedWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="previewPage">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="previewLabel">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="text">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="noPreviewPage">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="noPreviewLabel">
                  <property name="text">
                   <string>No preview available</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="getPreviewButton">
                  <property name="text">
                   <string>Get Preview</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QnDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnTableView</class>
   <extends>QTableView</extends>
   <header location="global">ui/widgets/common/table_view.h</header>
  </customwidget>
  <customwidget>
   <class>QnDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header location="global">ui/widgets/common/dialog_button_box.h</header>
  </customwidget>
  <customwidget>
   <class>QnPtzTourWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/ptz_tour_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <connections/>
</ui>
